lib/action_subscriber/rspec.rb

```diff
@@ -1,5 +1,3 @@
-require 'rspec'
```

Comment:
Instead of removing this, let's require `rspec/core`. Better to be safe and explicit IMO. That's all the `rspec.rb` does anyway: https://github.com/rspec/rspec/blob/master/lib/rspec.rb

Thanks for reporting this, btw!

Reply:
Awesome! Thank you for taking a look at this so quickly. I really appreciate it. I made the requested change.